src/config.rs

```diff
@@ -480,6 +480,12 @@ pub struct CommandLineArgs {
     /// Show the reason for skipped steps
     #[clap(long = "show-skipped")]
     show_skipped: bool,
+
+    /// Tracing filter directives.
+    ///
+    /// See: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/struct.EnvFilter.html
+    #[clap(long, default_value = "info")]
+    pub trace: String,
```

Comment:
New CLI option. Should this be documented somewhere?

Reply:
IMO rather than having a different option, it is more user friendly to change verbose to a count u8 (and maybe add a -q --quiet option) and implement tracing filter matching on that: -vv = trace -v = debug (-q = warning -qq error).

This is a rather common pattern in CLI design

Reply:
Those are not the same feature; while this can be used to limit logging based on level (`info`, `trace`, `debug`, `warn`, etc.), it can also do a lot more complex filtering; you can restrict log messages to various levels by module name, by extra data present, and more. See the linked docs:

https://docs.rs/tracing-subscriber/latest/tracing_subscriber/struct.EnvFilter.html

This sort of fine-grained filtering is super helpful when debugging, and I'm not sure I want to disable it in favor of only being able to select the level.

Reply:
I see your point, it's indeed useful for debugging, but should a debug option be exposed to the CLI? 

Since the EnvFilter support the common `RUST_LOG` variable, I feel like having the CLI having level only filtering via standard and self explanatory options, but document that `RUST_LOG` environment variable is available for more advanced filtering would be suffisant.

Else the name `--trace` is not really intuitive, if you want to keep it then `--log-filter` might be more explanatory?